service/collector/collector.go

```diff
@@ -112,15 +113,14 @@ func (c *Collector) Describe(ch chan<- *prometheus.Desc) {
 		ch <- trustedAdvisorSupport
 	}
 
-	ch <- elbsDesc
```

Comment:
The ELB collector is gone here because there is its own standalone implementation which we configure to the new and shiny exporterkit collector set.